firmware/application/apps/ui_mictx.hpp

```diff
@@ -113,6 +113,7 @@ class MicTXView : public View {
     uint32_t va_level{40};
     uint32_t attack_ms{500};
     uint32_t decay_ms{1000};
+    size_t iq_phase_calibration_value{15};
```

Comment:
Consider adding this parameter to app_settings (somewhere around line 130) so the setting will be remembered.

Reply:
Thanks @NotherNgineer  for your help and today's support .  (and also to @Bernd for yesterday's advises about radio part ) .  
 
Finally with Mark's help , we  managed 
a-) to keep persistent memory that IQ phase CALIBRATION  adjustment , 
b-) and we also solved the correct initial transmission using last calibrated data even we are not editing that value . 
Now both things are correct. 
I will add more measurents pictures on above comments .